Contribute/index.md

```diff
@@ -11,7 +11,7 @@ ms.custom: external-contributor-guide
 
 # Microsoft Docs contributor guide overview
 
-Welcome to the [Microsoft Docs](https://learn.microsoft.com) contributor guide!
+Welcome to the Microsoft Docs contributor guide!
```

Comment:
Should this be Microsoft Learn contributor guide?

Reply:
At present time, the contributor guide title will stay the same.